ckanext-hdx_theme/docs/code_quality/linter.rst

```diff
@@ -0,0 +1,60 @@
+Code Quality
+============
+
+Linter Configuration
+++++++++++++++++++++
+This section outlines how to configure the linter, Ruff, for your local development environment in PyCharm.
+
+Installing Ruff Linter
+----------------------
+
+Since we utilize CKAN within a Docker container, you'll need Ruff installed locally on your development machine.
+
+Ruff is available as ``ruff`` on PyPI:
+
+::
+
+    pip install ruff
+
+Starting with version ``0.5.0``, Ruff can be installed using their standalone installers:
+
+::
+
+  # On macOS and Linux.
+  curl -LsSf https://astral.sh/ruff/install.sh | sh
+
+  # On Windows.
+  powershell -c "irm https://astral.sh/ruff/install.ps1 | iex"
+
+  # For a specific version.
+  curl -LsSf https://astral.sh/ruff/0.5.0/install.sh | sh
```

Comment:
Could we also add direct links to some of the ruff executables (for Linux, for Mac)